component/otelcol/receiver/receiver.go

```diff
@@ -99,14 +111,20 @@ func (r *Receiver) Update(args component.Arguments) error {
 		scheduler.WithHostExporters(rargs.Exporters()),
 	)
 
+	reg := prometheus.NewRegistry()
+	r.collector.Set(reg)
+
+	promExporter, err := sdkprometheus.New(sdkprometheus.WithRegisterer(reg), sdkprometheus.WithoutTargetInfo())
```

Comment:
Nitpick, but why is this WithoutTargetInfo? Is it to not pass unnecessary metrics to other prometheus.* components later on?